zp-relayer/utils/helpers.ts

```diff
@@ -157,3 +159,21 @@ export function withLoop<R>(f: () => Promise<R>, timeout: number, supressedError
     }
   }
 }
+
+export function waitForFunds(web3: Web3, address: string, cb: (balance: BN) => void, minimumBalance: BN) {
+  return promiseRetry(
```

Comment:
Are we using the default interval here? I think it makes sense to retry the request every minute.

Reply:
Updated to 1 minute